bears/general/ShebangBear.py

```diff
@@ -0,0 +1,33 @@
+import os
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class ShebangBear(LocalBear):
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Syntax'}
```

Comment:
Hmm not sure if it's syntax... do we have a better fitting keyword?

Reply:
You aren't correcting syntax errors... hmm... Could be that you have to create a new keyword, maybe `Shebang` itself works?
You could maybe get some ideas from the community ;)